tests/subsys/zigbee/osif/logger/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
Please update to use `find_package(Zephyr ...)`

Reply:
updated